intrinsics_tester/commandLineUtil.h

```diff
@@ -0,0 +1,151 @@
+#pragma once
+
+// Copyright (c) 2021-present The Alive2 Authors.
+// Distributed under the MIT license that can be found in the LICENSE file.
+// Version: April 7, 2022
+
+#include <algorithm>
+#include <cstdlib>
+#include <iostream>
+#include <ir/instr.h>
+#include <string>
+#include <vector>
+
+struct CommandLineUtil {
+  static bool useDebugMode;
+  static unsigned lowerInclusiveBound;
+  static unsigned upperExclusiveBound;
+  static unsigned numberOfRepetitions;
+
+  CommandLineUtil(int argc, char **argv) {
+    if (argc < 2) {
+      std::cerr << "ERROR: Needs atleast one argument. Add -h for help."
+                << std::endl;
+      exit(1);
+    }
+
+    for (int i = 1; i < argc; ++i) {
+      std::string arg(argv[i]);
+      // All args are made lowercase
+      std::transform(arg.begin(), arg.end(), arg.begin(), ::tolower);
```

Comment:
Let's assume users knew the arguments need to be case sensitive, and we'll treat those arguments with capital letters as illegal. So remove this line